<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.mai.lessons.rpks.ServiceTest" tests="18" skipped="0" failures="0" errors="0" timestamp="2023-06-22T13:40:10" hostname="MSI" time="277.87">
  <properties/>
  <testcase name="testKafkaWriteReadMessage()" classname="ru.mai.lessons.rpks.ServiceTest" time="15.927"/>
  <testcase name="testServiceFilteringNotEquals()" classname="ru.mai.lessons.rpks.ServiceTest" time="13.857"/>
  <testcase name="testServiceFilteringNotRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="22.586"/>
  <testcase name="testServiceFilteringContainsNotEqualsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.621"/>
  <testcase name="testServiceFilteringAddNewRule()" classname="ru.mai.lessons.rpks.ServiceTest" time="33.018"/>
  <testcase name="testServiceFilteringContainsNotContainsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="13.893"/>
  <testcase name="testStartPostgreSQL()" classname="ru.mai.lessons.rpks.ServiceTest" time="11.037"/>
  <testcase name="testServiceFilteringUpdateRule()" classname="ru.mai.lessons.rpks.ServiceTest" time="32.739"/>
  <testcase name="testServiceFilteringNotContains()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.501"/>
  <testcase name="testServiceFilteringEqualsNotContainsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.341"/>
  <testcase name="testServiceFilteringContains()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.518"/>
  <testcase name="testServiceFilteringEqualsContainsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.616"/>
  <testcase name="testServiceFilteringEquals()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.445"/>
  <testcase name="testPostgreSQLReadValues()" classname="ru.mai.lessons.rpks.ServiceTest" time="10.817"/>
  <testcase name="testServiceFilteringEqualsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.956"/>
  <testcase name="testServiceFilteringEqualsNotEqualsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="13.004"/>
  <testcase name="testServiceFilteringContainsTwoRules()" classname="ru.mai.lessons.rpks.ServiceTest" time="12.221"/>
  <testcase name="testStartKafka()" classname="ru.mai.lessons.rpks.ServiceTest" time="10.766"/>
  <system-out><![CDATA[16:40:09,831 |-INFO in ch.qos.logback.classic.LoggerContext[default] - This is logback-classic version 1.4.5
16:40:09,847 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
16:40:09,849 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [file:/D:/%d0%9f%d1%80%d0%be%d0%b3%d1%80%d0%b0%d0%bc%d0%bc%d1%8b/IntelliJ%20IDEA%20Community%20Edition%202020.3.2/JavaServiceFiltering1/build/resources/test/logback.xml]
16:40:09,850 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@4b45dcb8 - Resource [logback.xml] occurs multiple times on the classpath.
16:40:09,850 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@4b45dcb8 - Resource [logback.xml] occurs at [file:/D:/%d0%9f%d1%80%d0%be%d0%b3%d1%80%d0%b0%d0%bc%d0%bc%d1%8b/IntelliJ%20IDEA%20Community%20Edition%202020.3.2/JavaServiceFiltering1/build/resources/main/logback.xml]
16:40:09,850 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@4b45dcb8 - Resource [logback.xml] occurs at [file:/D:/%d0%9f%d1%80%d0%be%d0%b3%d1%80%d0%b0%d0%bc%d0%bc%d1%8b/IntelliJ%20IDEA%20Community%20Edition%202020.3.2/JavaServiceFiltering1/build/resources/test/logback.xml]
16:40:09,930 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [CONSOLE]
16:40:09,930 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
16:40:09,966 |-WARN in ch.qos.logback.core.ConsoleAppender[CONSOLE] - This appender no longer admits a layout as a sub-component, set an encoder instead.
16:40:09,966 |-WARN in ch.qos.logback.core.ConsoleAppender[CONSOLE] - To ensure compatibility, wrapping your layout in LayoutWrappingEncoder.
16:40:09,966 |-WARN in ch.qos.logback.core.ConsoleAppender[CONSOLE] - See also http://logback.qos.ch/codes.html#layoutInsteadOfEncoder for details
16:40:09,966 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting level of logger [ru.mai.lessons.rpks] to INFO
16:40:09,969 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting additivity of logger [ru.mai.lessons.rpks] to false
16:40:09,969 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [CONSOLE] to Logger[ru.mai.lessons.rpks]
16:40:09,970 |-INFO in ch.qos.logback.classic.model.processor.RootLoggerModelHandler - Setting level of ROOT logger to ERROR
16:40:09,970 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [CONSOLE] to Logger[ROOT]
16:40:09,970 |-INFO in ch.qos.logback.core.model.processor.DefaultProcessor@7216fb24 - End of configuration.
16:40:09,971 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@2072acb2 - Registering current configuration as safe fallback point

16:40:21.769 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Bootstrap.servers: PLAINTEXT://localhost:62375
16:40:21.773 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:40:21.970 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Creating topics
16:40:22.592 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Created topics
16:40:22.592 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Sending message
16:40:22.685 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Sent message
16:40:22.685 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Consumer subscribe
16:40:22.685 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Consumer start reading
16:40:22.795 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:40:23.913 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 9
16:40:24.924 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:40:36.287 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:40:37.046 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:40:37.422 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:40:37.447 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:40:37.718 [pool-2-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:40:37.988 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:40:37.995 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:40:37.996 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:40:37.998 [pool-2-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:40:37.999 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:40:38.000 [pool-2-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:40:38.765 [pool-2-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:40:38.766 [pool-2-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:40:49.669 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:40:49.690 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:40:50.283 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:40:50.361 [pool-4-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:40:50.364 [pool-4-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:40:51.394 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 9
16:40:52.520 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 8
16:40:53.634 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 7
16:40:54.757 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 6
16:40:55.876 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 5
16:40:56.990 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 4
16:40:58.114 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 3
16:40:59.239 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 2
16:41:00.356 [pool-4-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 1
16:41:01.360 [pool-4-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:41:12.252 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:41:12.280 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:41:12.673 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:41:12.690 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:41:12.693 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:41:12.720 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:41:12.964 [pool-6-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:41:13.024 [pool-6-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:41:13.025 [pool-6-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:13.025 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:41:13.026 [pool-6-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:13.027 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:41:13.027 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:13.028 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:41:13.028 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:13.029 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:41:13.029 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:41:13.029 [pool-6-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:41:13.969 [pool-6-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:41:13.969 [pool-6-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:41:17.582 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:41:25.171 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:41:25.203 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:41:25.601 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:41:25.619 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:41:25.847 [pool-8-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:41:25.899 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:25.900 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:25.900 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:25.901 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:25.902 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:25.902 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:25.902 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:41:25.903 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:41:26.861 [pool-8-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:41:26.862 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:41:26.865 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:41:26.867 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Wait until application updated rules from DB
16:41:40.097 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:41:46.878 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:46.882 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:46.888 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:46.895 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:46.902 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:46.907 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:46.913 [pool-8-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:41:46.919 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:41:46.919 [pool-8-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:41:46.927 [pool-8-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:41:46.928 [pool-8-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:41:47.595 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:41:52.784 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:41:57.965 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:41:57.982 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:41:58.349 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:41:58.363 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:41:58.365 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:41:58.397 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:41:58.615 [pool-10-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:41:58.673 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:41:58.674 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains true
16:41:58.681 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:58.682 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:58.682 [pool-10-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:41:58.683 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:41:58.683 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:41:58.683 [pool-10-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:41:58.684 [pool-10-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:41:59.738 [pool-10-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 9
16:42:00.771 [pool-10-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:42:00.771 [pool-10-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:42:10.111 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:17.609 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:22.601 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:42:22.621 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:42:22.790 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:23.010 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:42:23.024 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:42:23.268 [pool-13-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:42:23.324 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:42:23.329 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:42:23.330 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:42:23.331 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:42:23.332 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:42:23.332 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:42:23.332 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:42:23.333 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:42:24.301 [pool-13-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:42:24.308 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:42:24.311 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:42:24.313 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Wait until application updated rules from DB
16:42:25.663 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:38.433 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:40.119 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:44.320 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:42:44.323 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:42:44.327 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:42:44.332 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:42:44.336 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:42:44.341 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:42:44.344 [pool-13-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:42:44.348 [pool-13-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:42:44.353 [pool-13-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:42:44.353 [pool-13-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:42:47.610 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:52.793 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:55.556 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:42:55.573 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:42:55.675 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:42:55.971 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:42:55.986 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:42:56.201 [pool-15-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:42:56.283 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:42:56.284 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:42:56.284 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:42:56.285 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:42:56.285 [pool-15-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:42:56.286 [pool-15-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains true
16:42:57.226 [pool-15-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:42:57.226 [pool-15-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:07.930 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:07.958 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:08.343 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:08.356 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:08.358 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:08.391 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:08.440 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:08.601 [pool-17-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:08.643 [pool-17-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:08.644 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:08.644 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains false
16:43:08.645 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:08.645 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:43:08.646 [pool-17-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:43:08.646 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:43:08.647 [pool-17-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:08.647 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:08.648 [pool-17-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_contains true
16:43:09.614 [pool-17-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:09.615 [pool-17-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:10.124 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:17.612 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:20.401 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:20.418 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:20.778 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:20.790 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:21.002 [pool-19-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:21.087 [pool-19-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:21.088 [pool-19-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:21.088 [pool-19-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:21.088 [pool-19-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:43:21.088 [pool-19-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:43:22.032 [pool-19-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:22.032 [pool-19-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:22.807 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:23.085 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:25.681 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:32.938 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:32.953 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:33.324 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:33.340 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:33.342 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:33.361 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:33.574 [pool-21-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:33.623 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:33.623 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:43:33.630 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:43:33.631 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:33.631 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:33.631 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:33.631 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:33.631 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:33.632 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:33.632 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:43:33.633 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:33.633 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:43:33.633 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:33.633 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:43:33.634 [pool-21-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:33.634 [pool-21-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:43:34.579 [pool-21-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:34.580 [pool-21-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:36.031 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:38.442 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:40.134 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:45.423 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:43:45.439 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:43:45.876 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:45.890 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:43:46.082 [pool-23-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:43:46.148 [pool-23-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:43:46.149 [pool-23-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:43:46.149 [pool-23-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:43:46.150 [pool-23-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:43:46.150 [pool-23-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:43:47.121 [pool-23-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:43:47.121 [pool-23-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:43:47.621 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:48.433 [pool-18-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:52.808 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:53.088 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:55.685 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:43:57.900 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:43:57.912 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:00.831 [pool-20-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:06.044 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:08.456 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:09.085 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:44:09.101 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:44:09.475 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:09.494 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:09.496 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:09.532 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:09.758 [pool-26-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:44:09.801 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:09.802 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:09.803 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:09.804 [pool-26-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:09.804 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:44:09.804 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:44:09.804 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:44:09.805 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:44:09.805 [pool-26-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:44:10.145 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:10.793 [pool-26-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:44:10.794 [pool-26-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:44:13.404 [pool-22-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:17.624 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:18.446 [pool-18-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:22.298 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:44:22.314 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:44:22.710 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:22.727 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:22.730 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:22.763 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:22.822 [pool-7-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:22.991 [pool-28-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:44:23.020 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:23.021 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:44:23.021 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals false
16:44:23.021 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:23.022 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:23.023 [pool-28-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:23.023 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals false
16:44:23.024 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - equals true
16:44:23.024 [pool-28-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - not_equals true
16:44:23.100 [pool-14-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:24.027 [pool-28-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:44:24.027 [pool-28-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:44:25.692 [pool-9-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:25.926 [pool-24-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:30.844 [pool-20-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:34.412 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topics: [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)], replica factor 1, partitions 3
16:44:34.428 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Topic not exist. Create topics [(name=test_topic_in, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null), (name=test_topic_out, numPartitions=3, replicationFactor=1, replicasAssignments=null, configs=null)]
16:44:34.764 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:34.777 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:34.779 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Create filtering rule 1
16:44:34.797 [Test worker] INFO  ru.mai.lessons.rpks.ServiceTest - Check rule from DB
16:44:35.001 [pool-30-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Remaining attempts 10
16:44:35.064 [pool-30-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception json
16:44:35.065 [pool-30-thread-1] ERROR r.m.l.rpks.impl.MyRuleProcessor - exception null
16:44:35.066 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:35.066 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:35.066 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:35.066 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains false
16:44:35.066 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:35.066 [pool-30-thread-1] INFO  r.m.l.rpks.impl.MyRuleProcessor - contains true
16:44:36.024 [pool-30-thread-2] INFO  ru.mai.lessons.rpks.ServiceTest - Read messages 1
16:44:36.025 [pool-30-thread-1] ERROR r.m.lessons.rpks.impl.MyKafkaReader - Create consumer exception
org.apache.kafka.common.errors.InterruptException: java.lang.InterruptedException
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.maybeThrowInterruptException(ConsumerNetworkClient.java:535)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:296)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:251)
	at org.apache.kafka.clients.consumer.KafkaConsumer.pollForFetches(KafkaConsumer.java:1307)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1243)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1216)
	at ru.mai.lessons.rpks.impl.MyKafkaReader.processing(MyKafkaReader.java:54)
	at ru.mai.lessons.rpks.impl.ServiceFiltering.start(ServiceFiltering.java:38)
	at ru.mai.lessons.rpks.ServiceTest.lambda$testStartService$49(ServiceTest.java:1332)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InterruptedException: null
	... 13 common frames omitted
16:44:36.048 [pool-16-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:38.466 [pool-11-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:40.145 [pool-5-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:43.416 [pool-22-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
16:44:47.634 [pool-3-thread-1] ERROR ru.mai.lessons.rpks.impl.MyDbReader - Connection dataSource exception
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
